### PR TITLE
completed html-forms exercise

### PR DESCRIPTION
**Pull request by hollyphung95**
Kapture 2023-01-24 at 14 03 28

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <div>
+        <input type="email" name="email" id="email">
+      </div>
+    </div>
+
+    <div>
+      <label for="why-code">Why is coding cool?</label>
+      <div>
+        <select id="why-code" name="reason">
+          <option value=""> Select an option</option>
```
Comment (robgmerrill) [inline html-forms/index.html:22]: small nitpick but there is an empty space before the text "Select"

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          <option value="interesting">Because it's interesting</option>
```
Comment (robgmerrill) [inline html-forms/index.html:23]: Please use HTML entity `&apos;` instead of '`'. Lines 24 and 25 as well.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great job! Just pointed out a few small things to fix.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Looks like we still have the requested changes on lines 23, 24, 25.

File: html-forms/index.html (hunk 1 above)
Comment (hollyphung95) [reply html-forms/index.html]: Ooops, im so sorry about that. Ive removed the leading space in line 22.

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**